library/cc/engine.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+// NOLINT(namespace-envoy)
```

Comment:
Let's namespace everything in these files. `Envoy::Mobile`, `Envoy::Platform`, `Envoy::Lib/rary` all seem like reasonable candidates to me.